ext/rugged/rugged_rebase.c

```diff
@@ -0,0 +1,273 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 GitHub, Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "rugged.h"
+
+extern VALUE rb_mRugged;
+extern VALUE rb_cRuggedTagAnnotation;
+extern VALUE rb_cRuggedTree;
+extern VALUE rb_cRuggedCommit;
+extern VALUE rb_cRuggedBlob;
+extern VALUE rb_cRuggedRepo;
+
+VALUE rb_cRuggedRebase;
+VALUE rb_cRuggedRebaseOperation;
+
+VALUE ruged_rebase_operation_new(git_rebase_operation *operation, VALUE owner);
+
+static void parse_rebase_options(git_rebase_options *ret, VALUE rb_options)
+{
+	VALUE val;
+
+	if (NIL_P(rb_options))
+		return;
+
+	val = rb_hash_aref(rb_options, CSTR2SYM("quiet"));
+	if (RTEST(val))
+		ret->quiet = 1;
+
+	val = rb_hash_aref(rb_options, CSTR2SYM("inmemory"));
+	if (!RTEST(val))
```

Comment:
I think `RTEST` returns `1` for truthy values and `0` for falsy ones, so you could change this (and similar code) to just: `ret->inmemory = RTEST(rb_hash_aref(rb_options, CSTR2SYM("inmemory")));`.